lib/src/chrome.dart

```diff
@@ -65,14 +65,25 @@ class Chrome {
   /// Starts Chrome with the given arguments and a specific port.
   ///
   /// Each url in [urls] will be loaded in a separate tab.
+  ///
+  /// If [userDataDir] is null, a new temp directory will be
+  /// passed to chrome as a user data directory. Chrome will
+  /// start without sign in and with extensions disabled.
+  ///
+  /// If [userDataDir] is not null, it will be passed to chrome
```

Comment:
wrap null is "ticks" (`null`) – is nice!

Reply:
cool:)

Reply:
done.